downstream/modules/automation-hub/con-required-shared-filesystem.adoc

```diff
@@ -5,4 +5,6 @@
 
 = Required shared filesystem
 
-A high availability {HubName} requires you to have a shared file system, such as NFS, already installed in your environment. Before you run the {PlatformName} installer, verify that the `/var/lib/pulp` directory exists across your cluster as a result of your shared file system installation. The {PlatformName} installer will return an error if `/var/lib/pulp` is not detected in one of your nodes, causing your HA {HubName} setup to fail.
+A high availability {HubName} requires you to have a shared file system, such as NFS, already installed in your environment. 
+Before you run the {PlatformName} installer, verify that the `/var/lib/pulp` directory exists across your cluster as a result of your shared file system installation. 
```

Comment:
Is this accurate:

verify that the `/var/lib/pulp` directory exists across your cluster as a result of your shared file system installation  >

verify that you installed the`/var/lib/pulp` directory across your cluster as part of your shared file system installation.

Reply:
I would say yes.